utils/update_checkout/update-checkout-config.json

```diff
@@ -120,7 +122,8 @@
                 "swift-markdown": "main",
                 "swift-cmark-gfm": "gfm",
                 "swift-nio": "2.31.2",
-                "swift-nio-ssl": "2.15.0"
+                "swift-nio-ssl": "2.15.0",
+                "swift-experimental-string-processing": "main"
```

Comment:
We'll want to switch to a tag ASAP. That way we get integration testing as PRs updating the tag, but otherwise churn on `main` doesn't affect the Swift project.

Reply:
I agree and initially wanted to do that.  Let's decide on a tagging scheme and move towards that in a follow up PR.